static/js/i18n/id.json

```diff
@@ -0,0 +1,21 @@
+{
+  "page-title": "Google Code-in 2017 Pemimpin Saat Ini",
+  "last-updated": "Terakhir diperbarui",
+  "order-info": "Peserta utama untuk setiap organisasi (dan akun GitHub mereka, jika ada) terdaftar secara acak.",
+  "tasks-completed": "Tugas selesai",
+  "footer-text": "Google Code-in dan logonya adalah merek dagang terdaftar dari Google Inc.",
+  "orgs-no-listed": "Organisasi tanpa pemimpin yang terdaftar",
+  "passed": "$1% passed, approximately $2 before task claiming ends",
```

Comment:
if it isnt translated, it should be omitted, so that we can build checks for percentage translated, and tools to create translations.

does `jquery.i18n` not support fallback to English for missing messages?

If it doesnt, please dont include languages which you dont know your translation is 100% accurate and complete.  We can use http://hosted.weblate.org/ to manage the collection of translations and ensure they are complete before the file is available for the i18n layer.